src/content/docs/apis/intro-apis/new-relic-api-keys.mdx

```diff
@@ -176,6 +176,13 @@ You should treat your API keys securely, as you would passwords and other sensit
   * Instruct your team members to keep their user keys secure.
   * When members leave your organization, even if they're [basic users](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type), remove them from New Relic.
 
+## Copying keys from the UI [#copy-keys]
+
+When you go to [API keys UI page](https://one.newrelic.com/api-keys), you can copy existing API keys. Here are some details about the language used in the UI: 
+
+* **Copy key** versus **Copy key ID**: The first option copies the value of the key itself. In almost all cases, you'll use that. The second option copies the ID of the key, which is sometimes needed for referencing the key object via API.  
+* **Copy truncated key**: This option is available for keys that are not yours. It copies only a few digits of the key. 
```

Comment:
@zuluecho9 I'm still not clear about the value of copying a truncated key that belongs to somebody else. Could you clarify that?